src/SpectatorView.Unity/Assets/SpectatorView/Scripts/UI/MobileNetworkConfigurationVisual.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Net;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace Microsoft.MixedReality.SpectatorView
+{
+    public class MobileNetworkConfigurationVisual : MonoBehaviour,
```

Comment:
>MobileNetworkConfigurationVisual [](start = 17, length = 32)

For the visual appearance of the default visual, I think it would be good to ensure there's an opaque or nearly-opaque background behind all of the text. This way the text is accessible and legible regardless of the background video you're holding the phone in front of.